Tools/setup/ubuntu.sh

```diff
@@ -69,20 +71,35 @@ elif [[ "${UBUNTU_RELEASE}" == "20.04" ]]; then
 	echo "Ubuntu 20.04"
 fi
 
+VERBOSE_BAR="####################"
+echo
+echo $VERBOSE_BAR
+echo "#⚡️ Starting PX4 Dependency Installer for Ubuntu ${UBUNTU_RELEASE} (${INSTALL_ARCH})"
+echo "# Options:
+#
+#  - Install NuttX = ${INSTALL_NUTTX}
+#  - Install Java = ${INSTALL_JAVA}
+#  - Install Simulation = ${INSTALL_SIM}
+#  - Install RTPS = ${INSTALL_RTPS}"
+echo $VERBOSE_BAR
+echo
 
 echo
-echo "Installing PX4 general dependencies"
+echo $VERBOSE_BAR
+echo "🍻 Installing System Dependencies"
+echo $VERBOSE_BAR
+echo
 
 sudo apt-get update -y --quiet
 sudo DEBIAN_FRONTEND=noninteractive apt-get -y --quiet --no-install-recommends install \
-	astyle \
 	build-essential \
-	cmake \
-	cppcheck \
-	file \
 	g++ \
 	gcc \
 	gdb \
+	astyle \
+	cmake \
+	cppcheck \
+	file \
```

Comment:
Why un-alphebetize the list?

Reply:
Wasn't on purpose. This was the part that was the hardest to test, I can change back to an alphabetized list.